roles/openshift-applier/README.md

```diff
@@ -192,12 +192,15 @@ metadata:
 
 ### Filtering content based on tags
 
-The `openshift-applier` supports the use of tags in the inventory (see example above) to allow for filtering which content should be processed and not. The `filter_tags` variable/fact takes a comma separated list of tags that will be processed and only content with matching tags will be applied.
+The `openshift-applier` supports the use of tags in the inventory (see example above) to allow for filtering which content should be processed and not. The `filter_tags` variable/fact takes a comma separated list of tags that will be processed and only content with matching tags will be applied. Additionally, tags can be skipped in the same manner with the `exclude_tags` option. 
+
+**_NOTE:_** These two options are mutually exclusive.
 
 **_NOTE:_** Entries in the inventory without tags will not be processed when a valid list is supplied with the `filter_tags` option.
 
 ```
 filter_tags=tag1,tag2
+exclude_tags=tag1,tag2
```

Comment:
This is example violates the "rules" as the same tags should not be allowed in both - i.e.: this should be an error case. Please update `exclude_tags` to be `tag3,tag4`.